examples/demo-app/src/app.js

```diff
@@ -356,6 +356,7 @@ class App extends Component {
                   width={width}
                   height={height - (showBanner ? BannerHeight : 0)}
                   onSaveMap={this._isCloudStorageEnabled() && this._toggleCloudModal}
+                  onSaveToStorage={() => {}}
```

Comment:
what would be the difference between onSaveMap and onSaveToStorage?
It feels like they should perform the same operation, should we merge it?

we could use onSaveToStorage and pass it to cloud provider interaction

Reply:
for backward compatibility. `onSaveMap` is currently called to share map with URL (by dropbox), `onSaveToStorage` is the newly added API which could be called by Carto - provider with a storage option. It's important to keep both, and later rename `onSaveMap` to `onShareMapURL` so that apps using the onSaveMap won't bread. Also we provide 2 methods to save / share, with an URL or directly opload assets to a storage.